java/sage/Person.java

```diff
@@ -196,6 +235,14 @@ public boolean isPersonIdentical(Person testMe)
       for (int i = 0; i < awardNames.length; i++)
         if (awardNames[i] != testMe.awardNames[i])
           return false;
+      for (int i = 0; i < headshotUrls.length; i++)
+      {
+        if (headshotUrls[i].length != testMe.headshotUrls[i].length)
+          return false;
+        for (int j = 0; j < headshotUrls[i].length; j++)
+          if (headshotUrls[i][j] != testMe.headshotUrls[i][j])
```

Comment:
I'm confused here. Are we not testing for the array to be identical to the other array? If I change != to == then we are saying that the objects are identical then their images are different.